### PR TITLE
Readded package: Aurora.Aurora version v0.8.1

### PR DESCRIPTION
**Pull request by OfficialEsco**
Gets stuck installing VC++, waiting for microsoft/winget-cli#163

- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

image

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22900)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
.

**Comment by OfficialEsco** [comment]
> Gets stuck installing VC++, waiting for microsoft/winget-cli#163

This needs Blocking-Issue.

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 
> 2023-12-27 12:09:50.593 [CLI ] ShellExecute installer failed: 1 
> 2023-12-27 12:09:50.403   Exception message:                           Internal error: Cannot expand "pf64" constant on this version of Windows.

(Automated response - build 490.)

File: manifests/a/Aurora/Aurora/v0.8.1/Aurora.Aurora.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/a/Aurora/Aurora/v0.8.1/Aurora.Aurora.installer.yaml:19]:
```suggestion
  Architecture: x64
```
From the error, I'm guessing that this is a 64-bit application.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by OfficialEsco** [comment]
works on my device 👀
image

**Comment by stephengillie** [comment]
Hi @OfficialEsco,

This PR appears to modify 2 versions: `v0.8.1` and `0.8.1` - is this an artifact of how GitHub is displaying the PR? 
image

**Comment by OfficialEsco** [comment]
Yes, we're currently on a branch before Aurora got deleted for a reason, the original PR was for fixing the DisplayVersion so its correctly matching.
I could try to rebase if you wish, but in theory only the add's will be merged as the deleted already exist

**Review by Trenly — APPROVED** [review]

**Pull request merged**